extensions/aws2-ddb/runtime/src/doc/limitations.adoc

```diff
@@ -0,0 +1,3 @@
+Feature `Batch consumer` (see https://camel.apache.org/components/latest/aws2-ddbstream-component.html#_batch_consumer[documentation])
+is not available in quarkus aws2-ddb extension because of the issue
+https://issues.apache.org/jira/browse/CAMEL-16837[CAMEL-16837].
```

Comment:
```suggestion
The https://camel.apache.org/components/latest/aws2-ddbstream-component.html#_batch_consumer[batch consumer] feature is not available in Camel Quarkus AWS2 DynamoDB extension because of https://issues.apache.org/jira/browse/CAMEL-16837[CAMEL-16837].
```